src/Runner.Worker/Handlers/StepHost.cs

```diff
@@ -110,9 +110,9 @@ public string ResolvePathForStepHost(string path)
 
             // try to resolve path inside container if the request path is part of the mount volume
 #if OS_WINDOWS
-            if (Container.MountVolumes.Exists(x => path.StartsWith(x.SourceVolumePath, StringComparison.OrdinalIgnoreCase)))
+            if (Container.MountVolumes.Exists(x => x.SourceVolumePath != null && path.StartsWith(x.SourceVolumePath, StringComparison.OrdinalIgnoreCase)))
```

Comment:
will `string.isnullorempty` better?

Reply:
Probably yes. All strings would technically start with "" so if it were empty we would unnecessarily call Container.TranslateToContainerPath. SourcePath shouldnt be empty but its probably safer